next/13/replace-next-head-v2/index.ts

```diff
@@ -933,6 +934,24 @@ const collectedImportedIdentifiers = (sourceFile: SourceFile) => {
 	return result;
 };
 
+const resolveModuleName = (path: string, containingPath: string) => {
+	if (project === null) {
+		return null;
+	}
+
+	return (
+		tsmorph.ts.resolveModuleName(
```

Comment:
can you explain how that works? do you need the entire project to be loaded?

Reply:
yes, we need to attach all js/ts/jsx/tsx files to project, but i will search for a better solution